staging/src/k8s.io/apiserver/pkg/util/flowcontrol/bootstrap/default-objects.go

```diff
@@ -0,0 +1,275 @@
+var (
+	InitialPriorityLevelConfigurations = []*rmv1a1.PriorityLevelConfiguration{
+		DefaultPriorityLevelConfigurationExempt,
+		DefaultPriorityLevelConfigurationDefault,
+	}
+	InitialFlowSchemas = []*rmv1a1.FlowSchema{
+		DefaultFlowSchemaExempt,
+		DefaultFlowSchemaDefault,
+	}
+)
+
+// Initial PriorityLevelConfiguration objects
+var (
+	DefaultPriorityLevelConfigurationExempt = pl(
+		rmv1a1.PriorityLevelConfigurationNameExempt,
+		rmv1a1.PriorityLevelConfigurationSpec{Exempt: true},
+	)
+	DefaultPriorityLevelConfigurationDefault = pl(
+		"default",
+		rmv1a1.PriorityLevelConfigurationSpec{
+			AssuredConcurrencyShares: 100,
+			Queues:                   128,
+			HandSize:                 6,
+			QueueLengthLimit:         100,
+		})
+)
+
+// Initial FlowSchema objects
+var (
+	DefaultFlowSchemaExempt = fs(
+		"exempt",
+		rmv1a1.PriorityLevelConfigurationNameExempt,
+		1500, "",
+		rmv1a1.PolicyRuleWithSubjects{
+			Subjects: groups(user.SystemPrivilegedGroup),
+			Rule: resourceRule(
+				[]string{rmv1a1.VerbAll},
+				[]string{rmv1a1.APIGroupAll},
+				[]string{rmv1a1.ResourceAll},
+			)},
+		rmv1a1.PolicyRuleWithSubjects{
+			Subjects: groups(user.SystemPrivilegedGroup),
+			Rule: nonResourceRule(
+				[]string{rmv1a1.VerbAll},
+				[]string{rmv1a1.NonResourceAll},
+			)},
+	)
+	DefaultFlowSchemaDefault = fs(
+		"default",
+		"default",
+		10000, rmv1a1.FlowDistinguisherMethodByUserType,
+		rmv1a1.PolicyRuleWithSubjects{
+			Subjects: groups(user.AllUnauthenticated, user.AllAuthenticated),
+			Rule: resourceRule(
+				[]string{rmv1a1.VerbAll},
+				[]string{rmv1a1.APIGroupAll},
+				[]string{rmv1a1.ResourceAll},
+			)},
+		rmv1a1.PolicyRuleWithSubjects{
+			Subjects: groups(user.AllUnauthenticated, user.AllAuthenticated),
+			Rule: nonResourceRule(
+				[]string{rmv1a1.VerbAll},
+				[]string{rmv1a1.NonResourceAll},
+			)},
+	)
+)
+
+// PredefinedPriorityLevelConfigurations returns the instances of
+// the ones defined in the KEP
+func PredefinedPriorityLevelConfigurations() []*rmv1a1.PriorityLevelConfiguration {
+	return []*rmv1a1.PriorityLevelConfiguration{
+		DefaultPriorityLevelConfigurationExempt,
+		pl("system", rmv1a1.PriorityLevelConfigurationSpec{
+			AssuredConcurrencyShares: 30,
+			Queues:                   1,
+			QueueLengthLimit:         1000}),
+		pl("workload-high", rmv1a1.PriorityLevelConfigurationSpec{
+			AssuredConcurrencyShares: 30,
+			Queues:                   128,
+			HandSize:                 6,
+			QueueLengthLimit:         100}),
+		pl("workload-low", rmv1a1.PriorityLevelConfigurationSpec{
+			AssuredConcurrencyShares: 30,
+			Queues:                   128,
+			HandSize:                 6,
+			QueueLengthLimit:         100}),
+		DefaultPriorityLevelConfigurationDefault,
+	}
+}
+
+// PredefinedFlowSchemas returns instances of the FlowSchema
+// objects shown in the Example Configuration section of the KEP
+func PredefinedFlowSchemas() []*rmv1a1.FlowSchema {
```

Comment:
seeing a list like this, I do think that we'll want these controller driven.  At least for an "exists with at least these IDs".  apiserver controllers are naturally active/active.

@lavalamp I'm flexible on timing though.  I'd accept poststarthook for alpha, but require a controller for beta.

Reply:
I think the comment here belongs with the logic that uses this list rather than the code that produces this list.  And there already is a discussion of this topic at that place.